app/assets/javascripts/admin/addon/components/webhook-events.js

```diff
@@ -78,6 +100,17 @@ export default class WebhookEvents extends Component {
       this.pingEnabled = true;
     }
 
+    if (data.type === "redelivered") {
+      const event = this.events.find((e) => {
+        return e.id === data.web_hook_event.id;
+      });
+      event.response_body = data.web_hook_event.response_body;
+      event.response_headers = data.web_hook_event.response_headers;
+      event.status = data.web_hook_event.status;
+      event.set("redelivering", false);
```

Comment:
Hmmm, I wonder if instead of mixing and matching `.property = X` and `set("redelivering", false)` you can try 
```
event.setProperties({
  ....
  status: data.web_hook_event.status,
  redelivering: false
})
```